src/config/basket.ts

```
@@ -41,6 +41,56 @@ export const BASKET_WEIGHTS_SUM = Object.values(BASKET_WEIGHTS).reduce(
+    throw new Error(
+      `Cannot allocate exact basis-point basket weights from provided values; got base sum ${baseSum}`,
+    );
+  }
+
+  entries.sort((a, b) => b.remainder - a.remainder);
+  for (let i = 0; i < missing; i += 1) {
+    entries[i].floored += 1;
+  }
+
+  return new Map(
+    entries.map((entry) => [entry.currency, Number((entry.floored / 100).toFixed(2))]),
```

ESLint

[error] Replace `entry.currency,·Number((entry.floored·/·100).toFixed(2))` with `⏎······entry.currency,⏎······Number((entry.floored·/·100).toFixed(2)),⏎····`

(prettier/prettier)